### PR TITLE
Making GitHub RSiena more available

### PR DESCRIPTION
**Pull request by jhollway**
# Description

* Added pushrelease workflow for building and releasing latest versions
* Added details on how to install from Github to README

# Checklist:

- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] The package builds on my OS without issues (please add workstation details)
- [x] My changes generate no new warnings
- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (.R, NEWS.md)
- [ ] I have bumped the version by the appropriate increment in the DESCRIPTION file (major, minor, patch)

### CODE REVIEW

**Review by TomSnijders — APPROVED** [review]
Very good.

**Comment by jhollway** [comment]
Still need to describe the pull request, etc.

File: README.md [hunk 1]
```diff
@@ -30,6 +30,19 @@ Some or all of these networks can be two-mode networks. The restriction is that
...
+
+`remotes::install_github("snlab-nl/rsiena@develop")`
+
+Or, if you would prefer to install a compiled binary, please download the latest release from Github,
+unzip the file, and install the OS-appropriate binary, for example:
+
+`install.packages("~/Downloads/rsiena_AllOS/macOS/rsiena_1.2.3.tgz", repos = NULL)`
+
 Note that the main manual can still be found [here](http://www.stats.ox.ac.uk/~snijders/siena/RSiena_Manual.pdf),
 and the main website is still [here](http://www.stats.ox.ac.uk/~snijders/siena/) for the time being,
```
Comment (TomSnijders) [inline README.md:47]: I would make explicit steps for naive users, and for all OSes. Is a compiled binary available also for Unix?
Proposal: first give instructions for downloading binaries and installing from them; then give instructions for installing from source.

Comment (jhollway) [reply README.md]: Done, please see commit d7d2ac80b097481a8de80dd00de017435b0d4d10

**Pull request merged**